sources/packages/backend/libs/sims-db/src/entities/cas-supplier.model.ts

```diff
@@ -14,6 +21,20 @@ export class CASSupplier extends RecordDataModel {
   @PrimaryGeneratedColumn()
   id: number;
 
+  /**
+   * Reference to student id in students table.
+   */
+  @OneToOne(() => Student, {
+    eager: false,
+    cascade: ["insert", "update"],
```

Comment:
The insert should not be allowed in this relationship.
As information, the SIN having the cascade as `true` is a mistake.